backend/src/main/java/gov/cdc/usds/simplereport/api/model/accountrequest/AccountRequest.java

```diff
@@ -0,0 +1,365 @@
+package gov.cdc.usds.simplereport.api.model.accountrequest;
+
+import static gov.cdc.usds.simplereport.api.Translators.sanitize;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import javax.validation.constraints.NotNull;
+
+@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
+public class AccountRequest {
+  @NotNull private String firstName;
+  @NotNull private String lastName;
+  @NotNull private String email;
+  @NotNull private String workPhoneNumber;
+  private String cellPhoneNumber;
+  @NotNull private String mailingAddress1;
+  private String aptSuiteOther;
+  private String aptFloorSuiteNo;
+  @NotNull private String city;
+  @NotNull private String state;
+  @NotNull private String zip;
+  @NotNull private String county;
+  @NotNull private String facilityType;
+  private String organizationName;
+  @NotNull private String facilityName;
+  @NotNull private String cliaNumber;
+  @NotNull private String testingDevices;
+  @NotNull private String accessDevices;
+  @NotNull private String browsers;
+  private String workflow;
+  @NotNull private String recordsTestResults;
+  @NotNull private String processTime;
+  @NotNull private String submittingResultsTime;
+  private String opFirstName;
+  private String opLastName;
+  private String npi;
+  private String opPhoneNumber;
+  private String opMailingAddress1;
+  private String opAptSuiteOther;
+  private String opAptFloorSuiteNo;
+  private String opCity;
+  private String opState;
+  private String opZip;
+  private String opCounty;
+
+  public String generateEmailBody() {
```

Comment:
This is great for a V1. Long term I think we should be using a template engine to manage emails sent from our backend. I added a ticket to track this work 


https://app.zenhub.com/workspaces/simplereport-5ff8be8854ae8b000e378210/issues/cdcgov/prime-simplereport/1086